CHANGELOG.md

```diff
@@ -19,6 +19,9 @@
 * Stabilize Web Share API.
   [#3882](https://github.com/rustwasm/wasm-bindgen/pull/3882)
 
+* Don't use reflect in WebIDL dictionary setters. Also, importing getters/setters from JS now supports names containing hyphens by optionally surrounding the name with quotes, e.g. `#[wasm_bindgen(method, setter = "x-cdm-codecs")]`.
+  [#3898](https://github.com/rustwasm/wasm-bindgen/pull/3898)
```

Comment:
I think the point here is not supporting hyphens, but strings in general.
It would also be good to mention that Web API bindings are going to be larger now.

Reply:
Looking at the generated code for `#[wasm_bindgen(method, setter = "x-cdm-codecs")]` I'm realising I really don't understand how it works.. I was never able to find the generated JS code that calls the setter or assigns the value even when closely inspecting a dummy project that makes use of it. But it somehow magically works :shrug: 

Anyways, I updated the changelog as requested